src/pages/Onboard/NameTeam.vue

```diff
@@ -11,14 +11,14 @@ export default {
     return {
       // Form's computed height
       height: '0px',
-
       loading: 0,
       accepting: false,
       activeInvite: null,
       currentInvitation: null,
       declining: false,
       dialog: false,
       redirectTenant: null,
+      pendingInvitations: [],
```

Comment:
We shouldn't need this since `pendingInvitations` will be moved to the top of the component by apollo anyway (same with that in the sidenav, now that the extra logic is removed)

Reply:
That's what I thought... but then I needed to add in pendingInvitations && pendingInvitations.length wherever we v-if it....

Reply:
Ah yea, then you can modify this line like this:
```
 update: data => data.pendingInvitations ?? []
```